misfitapp/models.py

```diff
@@ -72,24 +72,38 @@ def import_all_from_misfit(cls, misfit, uid):
 
 @python_2_unicode_compatible
 class MisfitUser(models.Model):
-    user = models.ForeignKey(UserModel)
-    misfit_user_id = models.CharField(max_length=MAX_KEY_LEN, primary_key=True)
-    access_token = models.TextField()
-    last_update = models.DateTimeField(null=True, blank=True)
+    """ A user's Misfit credentials, allowing API access """
+    user = models.ForeignKey(UserModel, help_text='The user')
+    misfit_user_id = models.CharField(
+        max_length=MAX_KEY_LEN,
+        primary_key=True,
+        help_text='The Misfit user ID')
+    access_token = models.TextField(help_text='The OAuth access token')
+    last_update = models.DateTimeField(
+        null=True,
+        blank=True,
+        help_text=(
+            'The datetime when the misfit user was last updated (deprecated)'
+        ))
 
     def __str__(self):
         return self.user.get_username()
 
 
 @python_2_unicode_compatible
 class Summary(MisfitModel):
-    user = models.ForeignKey(UserModel)
-    date = models.DateField()
-    points = models.FloatField()
-    steps = models.IntegerField()
-    calories = models.FloatField()
-    activity_calories = models.FloatField()
-    distance = models.FloatField()
+    """
+    Misfit daily summary data point:
+    https://build.misfit.com/docs/cloudapi/api_references#summary
+    """
+    user = models.ForeignKey(UserModel, help_text="The summary's user")
+    date = models.DateField(help_text='The date of the summary')
+    points = models.FloatField(help_text='Points for the day')
+    steps = models.IntegerField(help_text='Steps for the day')
+    calories = models.FloatField(help_text='Calories for the day')
+    activity_calories = models.FloatField(
+        help_text='Activity calories for the day')
+    distance = models.FloatField(help_text='Distance travelled during the day')
```

Comment:
@grokcode Good call, the documentation doesn't specify but it's probably miles. That would fit well with the examples and it looks like distance on other API end points come in miles.